alpha/apps/kaltura/lib/storage/urlRecognizer/kKalturaUrlRecognizer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ *
+ * @package Core
+ * @subpackage model
+ */
+
+class kKalturaUrlRecognizer extends kUrlRecognizer
+{
+
+	/**
+	 * @var string
+	 */
+	protected $secret;
+
+	/**
+	 * @return string
+	 */
+	public function getSecret()
```

Comment:
Lets rename to key to align with tokenizer and move the ket / get / set to the base class.